### DISCUSSION TITLE
Other Icons than fluent UI?

### DISCUSSION THREAD

**Discussion by Far0n** [h2oai/wave#985] [Q&A]
Can other icons than the fluent UI ones be used for the **icon** argument that some components have? Custom uploaded icons seem not to work. What's about other font-typed icons like **font-awesome**?

**Comment by mturoci** [comment]
Should be possible, just need a deeper thought on the reasonable API. Will think it through and get back next week.

**Comment by lo5** [comment]
https://github.com/h2oai/wave/issues/580

**Comment by Far0n** [comment]
@mturoci it also seems that not all icons are registered. For instance, I can't use the following: "ProvisioningPackage" from https://uifabricicons.azurewebsites.net/. Console log shows: "The item [..] was used but never registered"

Why is that?

**Reply by vopani** [reply]
@Far0n That's the older link. The updated full list should be this one: https://developer.microsoft.com/en-us/fluentui#/styles/web/icons

**Reply by Far0n** [reply]
Oh, okay .. Thank you @vopani!
